src/components/SignIn.js

```diff
@@ -189,15 +160,13 @@ export default function SignIn() {
         </Box>
       </Box>
       {/* useLogin hook errors */}
-      {errors && (
-        <Snackbar
-          open={open}
-          autoHideDuration={6000}
-          onClose={handleClose}
-          message={errors.msg} // Use the error message from the errors object
-          action={action}
-        />
-      )}
+      <Snackbar
+        open={open}
+        autoHideDuration={6000}
+        onClose={handleClose}
+        message={errors.msg} // Use the error message from the errors object
```

Comment:
Since `errors` can be null (ie: when you first load the page, there's no errors yet) this is currently causing an error when you try to load the page

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/d67fbd5c-067d-4649-8b6f-0ec29c3ffad8)

This error is telling us that Javascript is trying to access `errors.msg` but it can't because `errors` is null.

We can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here to tell Javascript that in some cases this value will be null, in which case just return null instead of trying to access `.msg`: 

```
message={errors?.msg}
```